airbyte-integrations/connectors/source-avni/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,14 @@
+{
+  "subjects": {
+    "Last modified at":"2200-06-27T04:18:36.914Z"
```

Comment:
The key is not `audit` -> `Last modified at`?

Reply:
@marcosmarxm  
state is  {"Last modified at":"date"}
and in schema we have audit inside that we have Last modified at [here](https://github.com/airbytehq/airbyte/pull/28141/files#diff-e9a00f38d57cd6a7f31088a5b86598cc9866d36c5b81877358d883d40694adb6R83)

keeping cursor field to only "Last modified at" one accepatance was failing so I have created a list of cursor field.
Did You have any better  solution for that.